features/admin.actions.v1/constants/actions-constants.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentityAppsError } from "@wso2is/core/errors";
+import { AuthTypeDropdownOption, AuthenticationType } from "../models";
+
+/**
+ * Model for action type.
+ * This class will hold the values for the URL path and the API path.
+ */
+class ActionType {
```

Comment:
have this in a different file

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/d96314b591f6f6f33b709b0600c399b789e7fcef